packages/manager/modules/telecom-spare/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@ovh-ux/manager-telecom-spare",
+  "version": "0.0.0",
+  "license": "BSD-3-Clause",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "browser": "./dist/umd/telecom-spare.js",
+  "scripts": {
+    "build": "rollup -c --environment BUILD:production",
+    "dev": "rollup -c --environment BUILD:development",
+    "dev:watch": "yarn run dev --watch",
+    "prepare": "yarn run build",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-telecom-spare' --include-filtered-dependencies -- yarn run build",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-telecom-spare' --include-filtered-dependencies -- yarn run dev",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-telecom-spare' --include-filtered-dependencies -- yarn run dev:watch"
+  },
+  "dependencies": {
+    "@ovh-ux/ng-ovh-telecom-universe-components": "^3.0.9",
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
+    "lodash": "^4.17.15"
+  },
+  "devDependencies": {
+    "@ovh-ux/component-rollup-config": "^6.0.0"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-core": "^6.1.1",
+    "@ovh-ux/manager-telecom-styles": "^3.1.0",
+    "@ovh-ux/ng-ovh-contact": "^4.0.2",
+    "@ovh-ux/ng-ovh-contracts": "^3.0.0-beta.3",
+    "@ovh-ux/ng-ovh-mondial-relay": "^6.0.1",
+    "@ovh-ux/ng-ovh-simple-country-list": "^1.0.0",
+    "@ovh-ux/ng-ovh-telecom-universe-components": "^3.0.7",
+    "@ovh-ux/ng-translate-async-loader": "^2.0.0",
+    "@uirouter/angularjs": "^1.0.20",
+    "angular": "^1.7.8",
+    "angular-animate": "^1.7.8",
+    "angular-leaflet-directive": "0.8.5",
+    "angular-translate": "^2.18.1",
+    "angular-translate-loader-partial": "^2.18.1",
+    "angular-translate-loader-pluggable": "^1.3.1",
+    "angular-translate-storage-cookie": "^2.18.1",
+    "angular-translate-storage-local": "^2.18.1",
+    "angular-ui-bootstrap": "^2.5.6",
+    "bootstrap4": "twbs/bootstrap#v4.0.0",
+    "flag-icon-css": "^2.3.1",
+    "font-awesome": "^4.7.0",
+    "international-phone-number": "^0.0.11",
+    "intl-tel-input": "~5.1.0",
+    "leaflet": "0.7.x",
+    "moment": "^2.24.0",
+    "oclazyload": "^1.1.0",
+    "ovh-api-services": "^9.10.1",
+    "ovh-common-style": "^5.0.0",
+    "ovh-manager-webfont": "^1.2.0",
+    "ovh-ui-angular": "^3.6.1",
+    "ovh-ui-kit": "^2.33.3",
+    "ovh-ui-kit-bs": "^2.1.1",
+    "ui-select": "^0.19.4"
```

Comment:
Do you really need all those peers ? Or just copy/paste from some other module?

Reply:
yes I need them, I added one after one during my development process.